adjust/kpis_test.go

```diff
@@ -51,16 +26,30 @@ func TestKPIService_ListWithOptions(t *testing.T) {
 		Sandbox:           false,
 		HumanReadableKpis: true,
 		Grouping:          "day,networks,campaigns,adgroups,creatives",
-		Reattributed:      "all",
-	}
-	got, _, err := client.KPI.List(context.Background(), &opt)
-	if err != nil {
-		t.Errorf("KPI.List returned error: %v", err)
-	}
+		Reattributed:      "all"},
+	},
+}
 
-	want := &KPI{}
-	responseToInterface(loadFixture("kpis_options.json"), &want)
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("KPI.List = %+v, want %+v", got, want)
+func TestKPIService_List(t *testing.T) {
+	for _, tt := range kpiTest {
```

Comment:
Would use tc (testcase) instead of tt. There seems to be different patterns, but imo tc makes more sense.
Like here: https://github.com/golang/go/wiki/TableDrivenTests